ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/PaywallAction.kt

```diff
@@ -0,0 +1,9 @@
+package com.revenuecat.purchases.ui.revenuecatui.components
+
+import com.revenuecat.purchases.paywalls.components.ButtonComponent
+
+internal sealed interface PaywallAction {
```

Comment:
Should we mark it with `@InternalRevenueCatAPI` for Java users? Honestly not sure, as Java users would already get a warning about using `internal` types.

Reply:
Hmm yeah... IMO, I think it would just be another warning they need to bypass but wouldn't actually remove the API for them... So I think I would probably leave it as is.

Reply:
Yea makes sense!